R/utils.R

```diff
@@ -39,8 +37,53 @@ epidist_version_stanvar <- function() {
 #' @param x A number to be rounded down
 #' @param f A positive number specifying the multiple to be rounded down to
 #' @importFrom checkmate assert_numeric
-#' @family utils
+#' @noRd
 floor_mult <- function(x, f = 1) {
   checkmate::assert_numeric(f, lower = 0)
   ifelse(f == 0, x, floor(x / f) * f)
 }
+
+#' Replace `brms` prior distributions
+#'
+#' This function takes `old_prior` and replaces any prior distributions
+#' contained in it by the corresponding prior distribution in `new_prior`.
+#' If there is a prior distribution in `new_prior` with no match in `old_prior`
+#' then the function will error and give the name of the new prior distribution
+#' with no match.
+#'
+#' @param old_prior One or more prior distributions in the class `brmsprior`
+#' @param new_prior One or more prior distributions in the class `brmsprior`
+#' @importFrom cli cli_inform
+#' @importFrom utils capture.output
```

Comment:
yourt missing the dplyr functions from importFrom

Reply:
[19104b3](https://github.com/epinowcast/epidist/pull/300/commits/19104b3f08cce359d07c2811815ed17bc8b2bd15)